tests/jenkins/set_user_permissions.sh

```diff
@@ -20,16 +17,19 @@ USER_NAME=$2
 GROUP_ID=$3
 GROUP_NAME=$4
 SCRIPT=$5
- 
+
 if [ -d “/usr/local/lib/“ ]; then
   chown -R ${USER_NAME}:${GROUP_NAME} /usr/local/lib/
 fi
 if [ -d “/usr/local/lib64/“ ]; then
   chown -R ${USER_NAME}:${GROUP_NAME} /usr/local/lib64/
 fi
-if [ -d “usr/local/bin/“ ]; then
+if [ -d “/usr/local/bin/“ ]; then
   chown -R ${USER_NAME}:${GROUP_NAME} /usr/local/bin/
 fi
 if [ -d “/opt/lib/“ ]; then
   chown -R ${USER_NAME}:${GROUP_NAME} /opt/lib/
 fi
+
+# Call run_as_user.sh script to set basic permission set.
+bash -c tests/jenkins/run_as_user.sh $1 $2 $3 $4 $5
```

Comment:
nit: variables GROUP_ID and SCRIPT are set above but aren't used. It might make this line more readable if we use the variables here instead of argument positions